### PR TITLE
[FIRRTL LowerTypes] Lower wire bundles to individual wires

### PR DESCRIPTION
**Pull request by youngar**

### CODE REVIEW

File: test/Dialect/FIRRTL/lower-types.mlir [hunk 1]
```diff
@@ -295,12 +295,12 @@ module  {
...
-//CHECK-NEXT: firrtl.connect %b_a, %x_a : !firrtl.flip<uint<1>>, !firrtl.uint<1>
+firrtl.circuit "RegBundle" {
+    // CHECK-LABEL: firrtl.module @RegBundle(%a_a: !firrtl.uint<1>, %clk: !firrtl.clock, %b_a: !firrtl.flip<uint<1>>) {
     firrtl.module @RegBundle(%a: !firrtl.bundle<a: uint<1>>, %clk: !firrtl.clock, %b: !firrtl.flip<bundle<a: uint<1>>>) {
+      // CHECK-NEXT: %x_a = firrtl.reg %clk {name = "x_a"} : (!firrtl.clock) -> !firrtl.uint<1>
+      // CHECK-NEXT: firrtl.connect %x_a, %a_a : !firrtl.uint<1>, !firrtl.uint<1>
+      // CHECK-NEXT: firrtl.connect %b_a, %x_a : !firrtl.flip<uint<1>>, !firrtl.uint<1>
       %x = firrtl.reg %clk {name = "x"} : (!firrtl.clock) -> !firrtl.bundle<a: uint<1>>
       %0 = firrtl.subfield %x("a") : (!firrtl.bundle<a: uint<1>>) -> !firrtl.uint<1>
       %1 = firrtl.subfield %a("a") : (!firrtl.bundle<a: uint<1>>) -> !firrtl.uint<1>
```
Comment (seldridge) [inline test/Dialect/FIRRTL/lower-types.mlir:306]: Feel free to rip NFC updates like this out and apply them directly onto `main`.

**Review by seldridge — APPROVED** [review]
lgtm

Nice work, @youngar!

**Comment by youngar** [comment]
Looks like the github action testing glitched out and never started.

**Comment by seldridge** [comment]
🤷‍♀️ I have no idea what GitHub actions are doing... (No issues reported on GitHub status page either.) I tried cancelling the workflow, but it won't cancel and I can't seem to restart it. The action disappeared from the PR, but it's stuck here: https://github.com/llvm/circt/actions/runs/610033121. 

@youngar: Maybe try doing a `git commit --amend --no-edit` and force pushing?

**Pull request merged**